src/tools/glsl.macro.js

```diff
@@ -0,0 +1,40 @@
+const { createMacro, MacroError } = require('babel-plugin-macros');
```

Comment:
nit: Can this be `import`s?

Reply:
Unfortunately it's not easy to get this code to run as an ES6 module. It's running in node as a Babel plugin and I can't figure out how to make it allow that.

Reply:
Ah yes, makes sense.